vtex/faststore
# Chore:  Makes sure merge is made before CMS sync

## What's the purpose of this pull request?

- When running cli `faststore cms-sync`, runs `generate` to ensure that sections and content-types files are merged or check if anything else is needed before calling `cms-sync`.

- We were having some issues because the `mergeCMSFiles` were only called when running `yarn dev` in the application, and the users couldn't see the changes made on the repository when sync with CMS. To avoid issues like that, it would be nice if `faststore cms-sync` checked it beforehand.

Thanks @ArthurTriis1 , for helping out! 🚀 🚀 

***We have to update the documentation!!!!***

## How to test it?

You can test using this PR: https://github.com/vtex-sites/starter.store/pull/119
On this repository run `yarn cms-sync`

![image](https://github.com/vtex/faststore/assets/3356699/fcaca922-b37c-41b9-aa30-a87d35adb94a)


You should be able your CMS changes applied in the workspace, without running `yarn dev`.
